test/unit/validator.test.js

```diff
@@ -624,6 +624,32 @@ describe('VALIDATE FUNCTION TESTS ', function () {
       });
     });
 
+    it('Should correctly validate path varaible in collection that are part of URL itself and are ' +
+      'not present in $request.url.variable', function (done) {
+      let multiplePathVarSpec = fs.readFileSync(path.join(__dirname, VALIDATION_DATA_FOLDER_PATH +
+          '/multiplePathVarSpec.json'), 'utf-8'),
+        multiplePathVarCollection = fs.readFileSync(path.join(__dirname, VALIDATION_DATA_FOLDER_PATH +
+          '/multiplePathVarCollection.json'), 'utf-8'),
+        resultObj,
+        historyRequest = [],
+        options = {
+          detailedBlobValidation: true,
+          allowUrlPathVarMatching: true
```

Comment:
Can we add a test for when this is false and validation fails or is it already covered by default?

Reply:
@diyaeninan as that's default behavior, it's already covered [here](https://github.com/postmanlabs/openapi-to-postman/blob/18153fb8476c5cc9df5a7ddd5d5e15ce99444b0f/test/unit/validator.test.js#L620).